mlflow/store/analytics/__init__.py

```diff
@@ -0,0 +1,26 @@
+"""
+Analytics modules for MLflow store operations.
+
+This package contains analytical algorithms and computations that operate
+on MLflow tracking store data, such as trace correlation analysis.
+"""
+
+from mlflow.store.analytics.trace_correlation import (
+    JEFFREYS_PRIOR,
+    LiftResult,
+    NPMIResult,
+    TraceCorrelationCounts,
+    calculate_expected_and_lift,
+    calculate_npmi_from_counts,
+    calculate_smoothed_npmi,
+)
+
+__all__ = [
```

Comment:
Just thought to add these here in case we needed it for the cli. If not, I'll wipe this file.

Reply:
Seems nice to have these here :)